etc/profile-m-z/postman.profile

```diff
@@ -0,0 +1,30 @@
+# Firejail profile for postman
+# Description: API testing platform
+# This file is overwritten after every install/update
+# Persistent local customizations
+include postman.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/Postman
+noblacklist ${HOME}/Postman
+
+include allow-nodejs.inc
```

Comment:
```suggestion
```

Are you sure that this is actually needed?

This is not included by any other profile.

Electron programs are usually proprietary and ship their own runtime.